### PR TITLE
Complete contact API

### PR DESCRIPTION
**Pull request by mcg-web**
- Complete contact API using documentation http://doc.rpc.gandi.net/contact/reference.html
- refactor `$this->getGandi()->getClient()->contact` to method to stop repeating it in each method
- remove php doc since their don't provide any more information since we using php7 typeHint

### CODE REVIEW

File: src/Api/Contact.php [hunk 1]
```diff
@@ -13,38 +13,97 @@
 
 namespace Nexy\Gandi\Api;
 
+use fXmlRpc\Proxy;
+
 /**
  * @author Jérôme Pogeant <p-jerome@hotmail.fr>
+ *
```
Comment (soullivaneuh) [inline src/Api/Contact.php:20]: You may add your `@author` tag if you want. :+1:

**Review by soullivaneuh — CHANGES_REQUESTED** [review]
PHPDoc annotations are required for methods.

File: src/Api/Contact.php (hunk 1 above)
Comment (mcg-web) [reply src/Api/Contact.php:20]: I'm not a big fan of `@author` annotation, no offense, but thank you anyway :laughing:

**Review by soullivaneuh — CHANGES_REQUESTED** [review]
LGTM. Asked @Jerome1337 for a review.

**Review by soullivaneuh — APPROVED** [review]

**Review by Jerome1337 — APPROVED** [review]
Everything seems good

**Comment by soullivaneuh** [comment]
Thank you @mcg-web! :+1:

**Pull request merged**